topgun/k8s/worker_lifecycle_test.go

```diff
@@ -26,6 +26,12 @@ var _ = Describe("Worker lifecycle", func() {
 		RerunJob                  bool
 	}
 
+	AfterEach(func() {
+		helmDestroy(releaseName)
```

Comment:
can we use the cleanup function as we do here: https://github.com/concourse/concourse/blob/master/topgun/k8s/baggageclaim_drivers_test.go#L18-L20

Reply:
@cirocosta can you make these changes and then I can merge

Reply:
Opened issue to remove the sleeps https://github.com/concourse/concourse/issues/3795